### PR TITLE
Update "Enabling the Ruby profiler" to use the new `ddprofrb` command

### PR DESCRIPTION
**Pull request by ivoanjo**
### What does this PR do? What is the motivation?

This PR updates the "Enabling the Ruby profiler" page to use the `ddprofrb` command as a replacement for the `ddtracerb` command that was deprecated in https://github.com/DataDog/dd-trace-rb/pull/3501 .

The old command is still going to be kept around for a while longer, and emits a deprecation warning when used.

### Merge instructions

- [x] Please merge after reviewing

The dd-trace-rb release containing this change fix (1.21.0) has not yet been released, so I'm queuing this documentation change but ask that we don't merge it yet, and I'll come back and leave a note once the release is out.

### Additional notes

N/A

### CODE REVIEW

File: content/en/profiler/enabling/ruby.md
Comment (delner) [inline content/en/profiler/enabling/ruby.md:54]: Should this not reflect `1.21`? The version in which this change is introduced?

**Comment by delner** [comment]
This change has been merged to `master`, pending release in 1.21.

File: content/en/profiler/enabling/ruby.md
Comment (cswatt) [inline content/en/profiler/enabling/ruby.md:106]:
```suggestion
    If starting the application with `ddprofrb exec` is not an option (for example, when using the Phusion Passenger web server), you can alternatively start the profiler by adding the following to your application entry point (such as `config.ru`, for a web application):
```

**Review by cswatt — APPROVED** [review]
just a punctuation/style suggestion. approving, but will wait for notification before merging. adding a DO NOT MERGE label.

File: content/en/profiler/enabling/ruby.md
Comment (ivoanjo) [inline content/en/profiler/enabling/ruby.md:54]:
```suggestion
    gem 'ddtrace', '~> 1.21'
```

Comment (ivoanjo) [reply content/en/profiler/enabling/ruby.md]: Fixed in https://github.com/DataDog/documentation/pull/22037/commits/5cb29c05195ef3813bba130e385fe19c69cb17cf

**Comment by ivoanjo** [comment]
Version https://rubygems.org/gems/ddtrace/versions/1.21.0 is now out, this is ready for review/merging :)

**Review by brett0000FF — APPROVED** [review]

**Pull request merged**
